gnarly.rb

```diff
@@ -188,6 +189,10 @@ def limit_test_logging
   end
 end
 
+def remove_chromedriver_helper
+  gsub_file "Gemfile", /(\s+#.*\s+)gem 'chromedriver-helper'/, ""
+end
```

Comment:
Rails 6 [removed this](https://github.com/rails/rails/commit/895edbcf406acfe1684bd41a4e3fb901b2ad6d6c) in favor of `webdrivers`. Maybe I'm getting ahead of myself here with #143, but perhaps we can...not care about this?

Reply:
We can ignore it for now, but that would mean _also_ not caring about having failing CI, which is why this is here at all. We _could_ close this PR and postpone it until #143 gets completed?

Reply:
Ok, nevermind, I very much care about CI passing.

Reply:
Just out of curiosity, what is the CI error with this in there now?

Reply:
If you leave it in, you get this error:
```
Selenium::WebDriver::Error::SessionNotCreatedError:
session not created: This version of ChromeDriver only supports Chrome version 76
```

Pretty much every recommendation from the CircleCI forums was to remove `chromedriver-helper` from the Gemfile.